src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -198,6 +199,9 @@
     <Reference Include="Microsoft.VisualStudio.CallHierarchy.Package.Definitions.dll">
       <HintPath>$(DevEnvDir)\Microsoft.VisualStudio.CallHierarchy.Package.Definitions.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.Shell.Design.dll">
+      <HintPath>$(DevEnvDir)\Microsoft.VisualStudio.Shell.Design.dll</HintPath>
```

Comment:
Is this a stable part of the VS SDK?  Do we need to hint-path it from here, or can it be referenced from the SDK?